docs/ad-hoc-tools/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Did you mean to change these files? I would expect only `reference/` changes in this PR from the title/description

Reply:
Huh, I don't know why/how these files where changed. Will fix.